### PR TITLE
Improve documentation around installation of AC Tool

### LINKED ISSUE #606: Improve documentation on how to install the AC Tool
the documentation around how to install the AC Tool best is quite slim (and misleading at the moment).

### PR DESCRIPTION
**Pull request by ghenzler**
Closes #606

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,69 +31,33 @@ It is also possible to run the AC Tool on **Apache Sling 11** or above (ensure s
...
-    <artifactId>accesscontroltool-package</artifactId>
-    <classifier>cloud</classifier>
-```
-(for AEM as a Cloud Service)
+* Install the latest version via AEM's package manager
+* Create a sample YAML file in CRXDE (e.g. `/apps/actool-test/test.yaml`)
+* Apply this config using the UI (see [User Interface](#user-interface) below
 
-Install it afterwards e.g. via AEM's package manager.
+For properly integrating the AC Tool in the development process see [Installation](docs/Installation.md).
```
Comment (kwin) [inline README.md]: the development process -> your own deployment package

File: README.md [hunk 2]
```diff
@@ -31,69 +31,33 @@ It is also possible to run the AC Tool on **Apache Sling 11** or above (ensure s
...
 
 You need to setup [Yaml configuration files](docs/Configuration.md) to specify your users, groups and ACL entries. See also the [best practices](docs/BestPractices.md) for hints on structuring.
 
 There are also some [advanced configuration options](docs/AdvancedFeatures.md) supported such as loops, conditional statements and permissions for anonymous.
 
-# Applying the ACL entries
+# User Interface
 
-There are multiple options to [apply the ACL entries](docs/ApplyConfig.md) (e.g. install hook, JMX and upload listener) to your target system.
+There is a Felix Web Console plugin (at `/system/console/actool`) as well as a Touch UI console (at `/mnt/overlay/netcentric/actool/content/overview.html`) to apply configurations and to inspect previous executions of the tool. Additionally there is a [JMX interface](docs/Jmx.md) interface for some advanced use cases.
```
Comment (kwin) [inline README.md]:
```suggestion
There is a Felix Web Console plugin (at `/system/console/actool`) as well as a Touch UI console (at `/mnt/overlay/netcentric/actool/content/overview.html`) to apply configurations and to inspect previous executions of the tool. Additionally there is a [JMX interface](docs/Jmx.md) for some advanced use cases.
```

File: docs/Installation.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
+# Installation
+
+The AC Tool [content package](https://jackrabbit.apache.org/filevault) contains the tool itself and is available from [the Maven Central repository](https://repo1.maven.org/maven2/biz/netcentric/cq/tools/accesscontroltool/accesscontroltool-package/). 
+
+In addition, there needs to be another package containing the project/platform specific [AC Tool configuration](Configuration.md) which is typically part of the regular project source code and released along with it (e.g. as `myproject-permissions`).
+
+Due to the different nature of on-premise and cloud service infrastructure setup, the maven setup for installation of the two packages is slightly different (while the AC Tool configuration as such is compatible to both and can usually easily be lifted from on-premise to the cloud service).
```
Comment (kwin) [inline docs/Installation.md]:
```suggestion
Due to the different nature of on-premise and cloud service infrastructure setup, the Maven setup for installation of the two packages is slightly different (while the AC Tool configuration as such is compatible with both setups and can usually easily be lifted from on-premise to the cloud service).
```

File: docs/Installation.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+## Installation AEM On-Premise
+
+For AEM 6.4/6.5 use the following appraoch:
```
Comment (kwin) [inline docs/Installation.md]:
```suggestion
For AEM 6.4/6.5 use the following approach:
```

File: docs/Installation.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+* Use a complete package that contains all project-related packages as sub package (hence in `/etc/packages`)
```
Comment (kwin) [inline docs/Installation.md]:
```suggestion
* Use a [container package](https://jackrabbit.apache.org/filevault/packagetypes.html) that contains all project-related packages as sub package (hence in `/etc/packages`)
```

File: docs/Installation.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+* Include the `accesscontroltool-package` *without* the `cloud` classifier (this has the [Sling Installer Provider Install Hook](https://sling.apache.org/documentation/bundles/installer-provider-installhook.html) in place)
```
Comment (kwin) [inline docs/Installation.md]:
```suggestion
* Include the `accesscontroltool-package` *without* the `cloud` classifier (this leverages the [Sling Installer Provider Install Hook](https://sling.apache.org/documentation/bundles/installer-provider-installhook.html))
```

File: docs/Installation.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+* Include the permissions package containing the project/platform specific [AC Tool configuration](Configuration.md)
+ * Ensure this permissions package declares the [dependency](#declaring-the-dependency) correctly
+
+
+Use the package with the GAV
```
Comment (kwin) [inline docs/Installation.md]: There is no V below. Rather use "Maven coordinates" or say "group id and artifact id"

File: docs/Installation.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+    <groupId>biz.netcentric.cq.tools.accesscontroltool</groupId>
+    <artifactId>accesscontroltool-package</artifactId>
+```
+
+without the `cloud` classifier.
+
+
+## Installation AEM Cloud Service
+
+For AEMaaCS use the use the following appraoch:
```
Comment (kwin) [inline docs/Installation.md]:
```suggestion
For AEMaaCS use the use the following approach:
```

File: docs/Installation.md [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+
+* Use an `all` package that contains all project-related packages as embedds (hence in `/apps`)
```
Comment (kwin) [inline docs/Installation.md]:
```suggestion
* Use a  container package that contains all project-related packages as embedded artifacts (hence in `/apps`)
```

File: docs/Installation.md [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+* Include the `accesscontroltool-package` *with* the `cloud` classifier (this contains the startup hook bundle for installation)
+* Include the permissions package containing the project/platform specific [AC Tool configuration](Configuration.md)
+ * Ensure this permissions package declares the [dependency](#declaring-the-dependency) correctly
+
+
+Use the package with the GAV and classifier `cloud`:
```
Comment (kwin) [inline docs/Installation.md]: Again no V (version) below

File: docs/Installation.md [hunk 11]
```diff
@@ -0,0 +1,112 @@
...
+(for AEM Classic/On Premise) or
+
+```
+    <groupId>biz.netcentric.cq.tools.accesscontroltool</groupId>
+    <artifactId>accesscontroltool-oakindex-package</artifactId>
+    <classifier>cloud</classifier>
+```
+(for AEM as a Cloud Service)
+
+Install it afterwards e.g. via AEM's package manager.
```
Comment (kwin) [inline docs/Installation.md]:
```suggestion
Install it afterwards e.g. via AEM's package manager or embed it in your container package.
```

**Review by kwin — CHANGES_REQUESTED** [review]

**Comment by ghenzler** [comment]
@kwin thanks for the review, I incorporated the suggestions in 47472bff

**Review by kwin — APPROVED** [review]

**Pull request merged**